pkg/provider/datasources/datasources.go

```diff
@@ -0,0 +1,62 @@
+package datasources
+
+type datasource string
```

Comment:
Rename to `DatasourceName`? I changed in usage tracking `resource` to `ResourceName`

Reply:
No. It is unexported on purpose.